### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kateryna3k**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int):
+        self.number = number
+
+    def movie_session(
```
Comment (wQuelS) [inline app/cinema/hall.py:8]: add type annotation to function.
If function returns nothing then it is -> None

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
 def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
```
Comment (wQuelS) [inline app/main.py:7]: same, type annotation

**Review by wQuelS — CHANGES_REQUESTED** [review]

**Review by wQuelS — APPROVED** [review]
Looks awesome imo:)

**Review by TPolina — APPROVED** [review]
Well done!

**Pull request closed without merging**